artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/QueueQueryImpl.java

```diff
@@ -95,70 +97,7 @@ public QueueQueryImpl(final boolean durable,
                          final SimpleString name,
                          final boolean exists,
                          final boolean autoCreateQueues) {
-      this(durable, temporary, consumerCount, messageCount, filterString, address, name, exists, autoCreateQueues, -1, false, false, RoutingType.MULTICAST);
```

Comment:
wont this be a breaking change removing the old api's if anyone is using CORE api directly? I appreciate its hidden for JMS users.

Reply:
Will fix.